buildscripts/kokoro/unix.sh

```diff
@@ -64,28 +64,17 @@ if [[ -z "${SKIP_TESTS:-}" ]]; then
   # --batch-mode reduces log spam
   mvn verify --batch-mode
   popd
-  pushd examples/example-alts
-  ../gradlew build $GRADLE_FLAGS
-  popd
-  pushd examples/example-hostname
-  ../gradlew build $GRADLE_FLAGS
-  mvn verify --batch-mode
-  popd
-  pushd examples/example-tls
-  ../gradlew build $GRADLE_FLAGS
-  mvn verify --batch-mode
-  popd
-  pushd examples/example-jwt-auth
-  ../gradlew build $GRADLE_FLAGS
-  mvn verify --batch-mode
-  popd
-  pushd examples/example-xds
-  ../gradlew build $GRADLE_FLAGS
-  popd
+  for f in `ls -d examples/example-*`
```

Comment:
You don't need to do `ls`. You can do a glob here to list folders: `for F in examples/example-*`